### PR TITLE
completed publish page layout

### PR DESCRIPTION
**Pull request by nazymberdaly**

### CODE REVIEW

**Review by mustafapylons — APPROVED** [review]
At Line 24, please change Max:10000 to reflect the update from Product Management for Max number of editions.
Otherwise it's good clean code.

File: lib/screens/home_screen.dart [hunk 1]
```diff
@@ -65,26 +66,60 @@ class _HomeScreenState extends State<HomeScreen> {
...
-                          onPressed: () {
-                            setState(() {
-                              _currentPage = _currentPage > 0 ? _currentPage - 1 : 0;
-                            });
+                    _currentPage != 3
+                        ? Padding(
+                            padding: const EdgeInsets.only(left: 10.0),
+                            child: IconButton(
+                                onPressed: () {
+                                  setState(() {
```
Comment (kjawadDeveloper2) [inline lib/screens/home_screen.dart]: Dont use setState

File: lib/screens/home_screen.dart [hunk 2]
```diff
@@ -65,26 +66,60 @@ class _HomeScreenState extends State<HomeScreen> {
...
+                                        fontSize: 20,
+                                        color: const Color(0xFF1212C4),
+                                        fontWeight: FontWeight.w400),
+                              ),
+                              IconButton(
+                                padding: EdgeInsets.all(0),
+                                  onPressed: () {},
+                                  icon: const Icon(
+                                    Icons.arrow_forward_ios,
+                                    color: Color(0xFF1212C4),
```
Comment (kjawadDeveloper2) [inline lib/screens/home_screen.dart]: Color should be provided through theme.

File: lib/widgets/rounded_purple_button_widget.dart [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+              color: const Color(0xFF1212C4).withOpacity(0.6),
+              offset: const Offset(0, 0),
+              blurRadius: 10.0)
+        ],
+        gradient: const LinearGradient(
+          begin: Alignment.topLeft,
+          end: Alignment.bottomRight,
+          stops: [0.0, 1.0],
+          colors: [
+            Color.fromRGBO(255, 255, 255, 0.2),
```
Comment (kjawadDeveloper2) [inline lib/widgets/rounded_purple_button_widget.dart]: Centralize all the colors.

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]
Kindly make these changes.

File: lib/screens/home_screen.dart (hunk 1 above)
Comment (nazymberdaly) [reply lib/screens/home_screen.dart]: okay

File: lib/screens/home_screen.dart (hunk 2 above)
Comment (nazymberdaly) [reply lib/screens/home_screen.dart]: okay

File: lib/widgets/rounded_purple_button_widget.dart (hunk 3 above)
Comment (nazymberdaly) [reply lib/widgets/rounded_purple_button_widget.dart]: okay

**Review by kjawadDeveloper2 — APPROVED** [review]
Looks Good from my side.

**Review by starhelios — APPROVED** [review]

**Pull request merged**